util/design/gen-otp-img.py

```diff
@@ -141,6 +141,22 @@ def main():
                         Note that seed values in additional configuration files
                         are ignored.
                         ''')
+    parser.add_argument('--data-perm',
+                        type=str,
+                        metavar='<map>',
+                        default='',
+                        help='''
+                        This is a post-processing option and allows to permute
```

Comment:
Nit: Should probably be "allows permuting"

Reply:
thanks, fixed.